packages/docs/src/components/layout/Layout.tsx

```diff
@@ -112,10 +112,10 @@ const Layout = ({
               {children}
             </article>
           ) : (
-            <article className="ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-padding-top--1 ds-u-padding-bottom--3 ds-u-md-display--flex page-body">
+            <article className="ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-padding-top--1 ds-u-padding-bottom--3 page-body">
               <div className="ds-l-row">
                 <div className="ds-l-lg-col--9">
-                  <div className="ds-u-display--block ds-u-lg-display--none ds-u-margin-bottom--3">
+                  <div className="ds-u-lg-display--none ds-u-margin-bottom--3">
```

Comment:
Removed this `ds-u-display--block` class because `div` is a display block by default and no other classes were changing this (until it was viewed at a large display, and that util makes sense)